build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -e
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+mkdir -p target
+
+mvn clean install -q -ff -DskipTests && docs/_bin/generate-license-dependency-reports.py . target --clean-maven-artifact-transfer && docs/_bin/generate-license.py licenses/APACHE2 licenses.yaml LICENSES.BINARY --dependency-reports target/license-reports
```

Comment:
since we are using "-Xmx3000m" in the next command anyway that means that much memory is available in this environment. so we can give this command same memory as well (not sure how much it needs but I am guess , less than 3000m)
so basically move "MAVEN_OPTS='-Xmx3000m" in a separate line before both the commands.

Reply:
Thanks. Moved it to top.